compatibility/bazel_tools/testing.bzl

```diff
@@ -435,6 +435,18 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        "start": "1.18.0-snapshot.20210928.7948.0.b4d00317",
+        "platform_ranges": [
+            {
+                "exclusions": [
+                    "KVCommandDeduplicationIT:KVCommandDeduplicationDeduplicateSubmitterBasic",
+                    "KVCommandDeduplicationIT:KVCommandDeduplicationSimpleDeduplicationBasic",
+                    "KVCommandDeduplicationIT:KVCommandDeduplicationCommitterDeduplication",
+                ],
+            },
+        ],
+    },
```

Comment:
What I'd like here is "don't cross-test the ledger API test tool V(snapshot after this PR is merged) with previous platform versions" but not sure how I can achieve this without a further change to this file after the next snapshot is released.

Reply:
Please make sure you get the above docs (starting at line 12), adding exclusions here is a bit tricky. The next snapshot version can be marked as `1.18.0-snapshot.20210928.7948.1`. So I think you'd like sth similar to what I did at line 396 for the gRPC error assertions.

Reply:
Yes, that's indeed the point; I think I get now that `1.18.0-snapshot.20210928.7948.1` is used not so much as "the next version" but rather as a fake version between the current and the next one to be used as a "start" without explicitly naming the next one (b/c we don't yet know its name). I'll try and improve the doc a bit.

Reply:
If you feel like changing this stuff so `"start"` is exclusive, not inclusive, it would probably simplify these compatibility checks. I think that's almost always what we want.